Python/sysmodule.c

```diff
@@ -2485,6 +2485,10 @@ sys_remote_exec_impl(PyObject *module, int pid, PyObject *script)
     PyObject *path;
     const char *debugger_script_path;
 
+    if (PySys_Audit("remote_exec", "iO", pid, script) < 0) {
```

Comment:
```suggestion
    if (PySys_Audit("sys.remote_exec", "iO", pid, script) < 0) {
```

For consistency with the rest of the module, and to reflect the actual operation that's taking place.

Reply:
Oh, and please move it after the `FSConverter` call. That way we only get the audit event for valid arguments.

Reply:
I'm not sure we need add the namespace for `remote_debugger_script` event at the same time

Reply:
No we don't, because that one isn't triggered by user code (as far as our current process is concerned).

Possibly it should be `cpython.remote_debugger_script` unless we're expecting the behaviour to be standard Python (which I doubt - `cpython.run_command`, `_module` and `_script` aren't assumed to be standard), but as it's already released (has it?) we can't change it (or shouldn't... if it's only been in beta then we can probably get away with it).

Reply:
It just beta. Update PR ing

Reply:
Done, PTAL